### DISCUSSION TITLE
Three dashes (---) in YAML files

### DISCUSSION THREAD

**Discussion by PWSchiffer** [WeblateOrg/weblate#10230] [Q&A]
Hi! We use Weblate to localize Ruby YAML files.
The source files always have "---" as a separator at the beginning. However, these characters are missing in the files generated by Weblate. This poses an unsolvable problem for our developers. Is it possible to configure Weblate to ensure that these characters are always available at the beginning?

**Comment by nijel** [comment]
This is currently not configurable, but it should be easy to add to https://docs.weblate.org/en/latest/admin/addons.html#customize-yaml-output
